src/components/app/app.test.ts

```diff
@@ -208,6 +209,14 @@ describe('app test suite', () => {
 
       expect(response.redirect).toEqual(orgs.definition.path);
     });
+
+    it('should add a meta tag for origin', async () => {
+      const orgs = router.findByName('admin.organizations');
+      const response = await agent.get(orgs.definition.path);
+
+      expect(response.status).toEqual(200);
+      expect(response.text).toMatch('<meta name="origin" content="uaa" />');
```

Comment:
I think `origin` might not be sufficiently specific for a meta tag (e.g. one might confuse it with "origin" as in "same origin policy") maybe `x-user-identity-origin` or something?

Reply:
Agreed, resolved in 75d7c63f323ae29cae6759f09afbd1004edf659d